maths/prime_number.rb

```diff
@@ -0,0 +1,31 @@
+# A ruby program to check a given number is prime or not
+# Mathematical explanation: A number which has only 2 factors i.e., 1 (one) and itself
+
+# Prime number check function
+def is_prime_number(number)
+  prime_flag = false
```

Comment:
**[Optional]**

Maybe it is a bit misleading that the number is a prime when `prime_flag` is `false`.